deployment/postgres-init-db/sql/tables/sequencing/sequence_adaptation.sql

```diff
@@ -1,15 +1,15 @@
 create table sequencing.sequence_adaptation (
   id integer generated always as identity,
   adaptation text not null,
-  name text not null,
+  name text not null default gen_random_uuid(),
```

Comment:
This lets users leave `name` blank while creating new `sequence_adaptation`s--is this desired?

Reply:
The UI blocks this behavior so I don't think this is a huge deal, if they upload directly through the API without providing a name I think a generated name is alright.